lib/index.js

```diff
@@ -73,50 +73,68 @@ function highlight (sqlString, options) {
       matches.push({
         name: hl.name,
         start: match.index + boringLength,
-        length: (hl.trimEnd ? text.substr(0, text.length - hl.trimEnd) : text).length
+        length: (hl.trimEnd
+          ? text.substring(0, text.length - hl.trimEnd)
+          : text
+        ).length
```

Comment:
Perhaps it'd be neater to break this out into it's own variable and use the length of that? `trimmedText.length` or smth.